app/views/events/edit.html.erb

```diff
@@ -1,2 +1,4 @@
-<h1>Edit event</h1>
-<%= render 'form', event: @event %>
+<div class="p-10">
+    <h1>Edit event</h1>
+    <%= render 'form', event: @event %>
+</div>
```

Comment:
Add an empty line at the end of your code

Reply:
fixed!!